packages/react-make-styles/src/makeStyles.ts

```diff
@@ -7,10 +7,6 @@ import { useRenderer } from './useRenderer';
 export function makeStyles<Slots extends string>(stylesBySlots: Record<Slots, MakeStylesStyleRule<Theme>>) {
   const getStyles = vanillaMakeStyles(stylesBySlots);
 
-  if (process.env.NODE_ENV === 'test') {
-    return () => ({} as Record<Slots, string>);
-  }
```

Comment:
Can you please remove the same condition for `__styles.ts` in the same directory?